### PR TITLE
V2 handler cleanup

### PR DESCRIPTION
**Pull request by jcorbin**
r @ShanniLi @kriskowal @Raynos

### CODE REVIEW

File: node/v2/handler.js [hunk 1]
```diff
@@ -700,25 +686,15 @@ TChannelV2Handler.prototype.sendCallRequestContFrame = function sendCallRequestC
         self.errorEvent.emit(self, errors.SendCallReqContBeforeIdentifiedError());
         return;
     }
-    var reqBody = new v2.CallRequestCont(flags, req.checksum.type, args);
-    var result = self._sendCallBodies(req.id, reqBody, req.checksum);
-    req.checksum = result.checksum;
 
-    var req0 = self.connection.ops.getOutReq(req.id);
```
Comment (jcorbin) [inline node/v2/handler.js]: @ShanniLi can you please explain why this was added in the first place?

Comment (ShanniLi) [inline node/v2/handler.js]: req is a request continue frame that doesn't have serviceName etc. req0 is the first request sent out that has the info.

**Comment by ShanniLi** [comment]
not sure we can drop the req lookup; otherwise, serviceName, cn, etc will be missing in stats for continuous requests.

File: node/v2/handler.js (hunk 1 above)
Comment (jcorbin) [inline node/v2/handler.js]: no, it's not a frame object, it's the outgoing request object, and should be identical (same object instance: req === req0).

**Comment by jcorbin** [comment]
Object identical, not just similar properties, but exact same object.

**Comment by kriskowal** [comment]
:ok:

**Pull request merged**